src/scrabble.js

```diff
@@ -1,5 +1,101 @@
-function scrabble() {
+const pointValuesForLetters = [
+  [1, 'A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T'],
+  [2, 'D', 'G'],
+  [3, 'B', 'C', 'M', 'P'],
+  [4, 'F', 'H', 'V', 'W', 'Y'],
+  [5, 'K'],
+  [8, 'J', 'X'],
+  [10, 'Q', 'Z']
+]
+let score = 0
 
+// // function takes any word and returns scrabble score
+function scrabble(word) {
+  // word to upper case
+  const givenWord = caseSensitive(word)
+  // UC word split to array of letters
+  const givenLetters = splitLetters(givenWord)
+  // for each of the given letters, loop ref. array
+  letterLookingFor(givenLetters)
 }
 
+// function turns all given letters to capitals
+const caseSensitive = (aString) => {
+  if (typeof aString === 'string') {
+    return aString.toUpperCase()
+  } else {
+    return ''
+  }
+}
+
+// split UC word into Array of Letters
+const splitLetters = (givenWord) => {
+  const givenLetters = givenWord.split('')
+  console.log('givenLetters =', givenLetters)
+  return givenLetters
+}
+
+// for each element of givenLetters, loop through reference array
+const letterLookingFor = (letterToCompare) => {
+  const questionedLetter5 = letterToCompare
+  letterToCompare.forEach(() => {
+    searchMainArray(pointValuesForLetters, questionedLetter5)
+  })
+}
+
+// look through pointsValuesForLetters array (loop)
+const searchMainArray = (mainArray, questionedLetter4) => {
+  const outerArray = mainArray
+  const questionedLetter3 = questionedLetter4
+  for (let i = 0; i <= outerArray.length - 1; i++) {
+    // each time - call function that searches that element (inner array)
+    searchInnerArray(outerArray[i], questionedLetter3)
+  }
+}
+
+// loop through elements of inner array, start idx 1
+const searchInnerArray = (subArray, questionedLetter) => {
+  const innerArray = subArray
+  const questionedLetter1 = questionedLetter
+  console.log(innerArray)
+  for (let j = 1; j <= innerArray.length - 1; j++) {
+    compareInnerArray(innerArray, questionedLetter1)
+  }
+}
+
+// compare given letter with inner array
+const compareInnerArray = (innerArrayElement, questionedLetter2) => {
+  const searchingList = innerArrayElement
+  console.log('searchingList =', searchingList)
+  const lookingFor = questionedLetter2
+  console.log('lookingFor =', lookingFor)
```

Comment:
Good use of console logging all over your code to gain visibility, it also sure does seem to make debugging this alot easier!